### PR TITLE
feat(Chat): Convert text into emojis on chatbar when settings for it is enabled

### LINKED ISSUE #528: implement(chats): convert to emoji is not working when is enabled
1. Create an account
2. Add any friend
3. Go to settings - messages
4. On Settings Messages, enable - Convert to Emoji feature
5. Start a conversation and type in chatbar input "<3" 

Expected Result: Texts like <3 should be converted to emoji (heart in this example) when Convert to Emoji is enabled

Actual Result: Texts like <3 are not converted to emoji (heart in this example) when Convert to Emoji is enabled

### PR DESCRIPTION
**Pull request by lgmarchi**
#### 1. Convert text into emojis on chatbar when settings for it is enabled, just for smileys and emotions

https://github.com/user-attachments/assets/b6da2b81-30f8-4b9b-bea0-29227c40027e

### Which issue(s) this PR fixes 🔨

- Resolve #528

### CODE REVIEW

**Comment by luisecm** [comment]
Working good for me. Updated the snapshots on testing repository and rerunning CI automated tests. PR approved from my side

**Review by luisecm — APPROVED** [review]

File: src/lib/components/messaging/emoji/category/activities.ts [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+    {
+        name: "video game",
+        glyph: "🎮",
+    },
+    {
+        name: "joystick",
+        glyph: "🕹",
+    },
+    {
+        name: "game die",
```
Comment (stavares843) [inline src/lib/components/messaging/emoji/category/activities.ts:255]:
```suggestion
        name: "game dice",
```

**Review by stavares843 — CHANGES_REQUESTED** [review]
when going npm run build appears this warning

Captura de ecrã 2024-09-11, às 14 48 08


it does not happen on dev

File: src/lib/components/messaging/emoji/category/foodAndDrink.ts [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+        name: "tangerine",
+        glyph: "🍊",
+    },
+    {
+        name: "lemon",
+        glyph: "🍋",
+    },
+    {
+        name: "⊛ lime",
+        glyph: "🍋‍🟩",
```
Comment (stavares843) [inline src/lib/components/messaging/emoji/category/foodAndDrink.ts:24]: is this on purpose? the green cube

**Review by InfamousVague — APPROVED** [review]

**Pull request merged**